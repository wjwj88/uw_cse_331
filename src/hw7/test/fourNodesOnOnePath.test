# Find a path with Dijkstra's algorithm when there is only one possible path with 4 nodes hw7

CreateGraph g
AddNode g A
AddNode g B
AddNode g C
AddNode g D
AddEdge g A	B 0.5	
AddEdge g B D 0.8 
AddEdge g D C 1.0

FindPath g A C