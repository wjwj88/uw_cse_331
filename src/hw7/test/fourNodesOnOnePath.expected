# Find a path with Dijkstra's algorithm when there is only one possible path with 4 nodes hw7

created graph g
added node A to g
added node B to g
added node C to g
added node D to g
added edge 0.500 from A to B in g
added edge 0.800 from B to D in g
added edge 1.000 from D to C in g

path from A to C:
A to B with weight 0.500
B to D with weight 0.800
D to C with weight 1.000
total cost: 2.300